src/awx_plugins/credentials/github_app.py

```diff
@@ -0,0 +1,142 @@
+# mypy: ignore-errors
+"""GitHub App Credential Plugin.
+
+This module defines a credential plugin for GitHub Apps, allowing
+authentication via GitHub App tokens.
+
+Functions:
+- github_app_backend: Generates a GitHub App token fields from GitHub Admin.
+- github_app_lookup: Defines the credential plugin interface.
+"""
+
+# # FIXME: the following violations must be addressed gradually and unignored
+# # mypy: disable-error-code="no-untyped-call, no-untyped-def"
```

Comment:
This was supposed to be at the top to work, plus it's a leftover from poorly maintained legacy code.